src/types.tsx

```diff
@@ -0,0 +1,12 @@
+export interface ErrorMessageType {
+  msg: string;
+  value?: string;
+  param?: string;
+}
+
+export interface Cohort {
+  _id: string;
+  name: string;
+  startDate: string;
+  endDate: string;
+}
```

Comment:
another option would be to have these types in a separate files instead of having everything in one file.

Reply:
we could create model.ts